labs/Language/Basics.md

```diff
@@ -2038,9 +2038,11 @@ lazyAbsMin x y = if x == 0 then 0 else min (abs x) (abs y)
 Does not produce an error when `x` is zero, regardless of the value of
 `y`. For instance:
 
-```Xcryptol-session
+```Xcryptol-session-ci-none
 labs::Language::Basics> lazyAbsMin 1 (0/0)
+```
 
+```Xcryptol-session
```

Comment:
I don't think that's in the right spot...perhaps down one line?

Reply:
There are a few parts of the form "Pretend you edit this file and then reload it; here's what happens".  I'm refactoring the ones I see and running the script locally until it passes.